importer/tests.py

```
@@ -1,3 +1,165 @@
+        # Mock directory contents
+        mock_file = MagicMock()
+        mock_file.name = "test_file.txt"
+        mock_file.is_dir.return_value = False
+        mock_input_path.iterdir.return_value = [mock_file]
+
+        mock_path_class.return_value = mock_input_path
+        mock_path_class.home.return_value = "/home/testuser"
+
+        response = self.client.get("/api/directories/", follow=True)
+
+        self.assertEqual(response.status_code, 200)
```

Ruff (0.15.0)

[warning] Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)